app/views/alert_requests/_form.html.erb

```diff
@@ -0,0 +1,93 @@
+<%= hidden_field_tag "partner", @partner_id %>
+
+<% content_for :head do %>
+<script>
+$(document).ready(function() {
+  $('#registrant_javascript_disabled').remove();
```

Comment:
I think this will need to change param name? And I think we'll need to track javascript_disabled as a boolean field on the alert_request model.